evalml/tests/preprocessing_tests/test_balanced_classification_sampler.py

```diff
@@ -82,15 +85,15 @@ def test_classification_imbalanced_balanced_ratio(num_classes, balanced_ratio):
     indices = bcs.fit_resample(X, y)
     X2 = X.loc[indices]
     y2 = y.loc[indices]
-    if balanced_ratio >= 3:
+    if balanced_ratio <= .3:
```

Comment:
This is because the default `balanced_ratio` is 0.25? Could you say `if balanced_ratio <= 0.25` for clarity?

(Its probably more trouble than its worth but you could also use introspection to retrieve the default `balanced_ratio` value from the method signature 😆 )

Reply:
@dsherry We choose 0.3 since the targets have a class ratio of 1/3 for the minority/majority balance. In scenarios where we consider anything less than 1/3 as balanced (for the sampler), we expect the data to not be undersampled. For instance, if we consider class ratios down to 1/5 as balanced, then this dataset shouldn't be sampled.